lightning/src/ln/channelmanager.rs

```diff
@@ -586,7 +589,8 @@ impl MsgHandleErrInternal {
 			err: LightningError { err, action },
 			chan_id: Some((channel_id, user_channel_id)),
 			shutdown_finish: Some((shutdown_res, channel_update)),
-			channel_capacity: Some(channel_capacity)
+			channel_capacity: Some(channel_capacity),
+			channel_funding_txo,
```

Comment:
nit: We can write this more verbosely to make it robust against variable name change.

```suggestion
			channel_funding_txo: channel_funding_txo,
```

Reply:
With field or parameter rename some adjustment is needed anyhow, but I get the additional felxibility of the suggestion. Accepted.